worker-languagedetection-container/pom.xml

```
@@ -293,7 +293,7 @@
                             <alias>worker-languagedetection</alias>
                             <name>${dockerCafDataProcessingOrg}worker-languagedetection${dockerProjectVersion}</name>
                             <build>
-                                <from>${dockerHubPublic}/cafapi/opensuse-jre11:3</from>
+                                <from>${dockerHubPublic}/cafapi/opensuse-jre17</from>
```

Our current policy if to reference the 1-part version number:

```suggestion
                                <from>${dockerHubPublic}/cafapi/opensuse-jre17:1</from>
```